src/typography/src/Heading.js

```diff
@@ -30,16 +40,7 @@ Heading.propTypes = {
   /**
    * The size of the heading.
    */
-  size: PropTypes.oneOf([100, 200, 300, 400, 500, 600, 700, 800, 900]),
-
-  /**
-   * Pass `default` to use the default margin top for that size.
-   */
-  marginTop: PropTypes.oneOfType([
```

Comment:
Worth making a note of this in the migration docs - I'll do that in #1020

Reply:
👍